enterprise/app/filter/filter_util.tsx

```diff
@@ -77,6 +79,60 @@ function splitAndTrimTags(param: string | null): string[] {
     .filter((s) => s);
 }
 
+function parseDimensionType(stringValue: string): stat_filter.Dimension | undefined {
+  if (stringValue.length < 2) {
+    return undefined;
+  }
+  const type = stringValue[0];
+  const enumValue = Number.parseInt(stringValue.substring(1));
+  if (!Number.isInteger(enumValue) || enumValue === 0) {
+    return undefined;
+  }
+  if (type == "e" && Object.values(stat_filter.ExecutionDimensionType).find((v) => v === enumValue)) {
+    return new stat_filter.Dimension({
+      execution: enumValue,
+    });
+  } else if (type == "i" && Object.values(stat_filter.InvocationDimensionType).find((v) => v === enumValue)) {
+    return new stat_filter.Dimension({
+      invocation: enumValue,
+    });
+  }
+  return undefined;
+}
+
+export function getFiltersFromDimensionParam(dimensionParamValue: string): stat_filter.DimensionFilter[] {
```

Comment:
nit: mind adding a comment with an example input string that this is parsing and what it returns?

Reply:
yup, done